normandy/recipes/api/serializers.py

```diff
@@ -1,13 +1,12 @@
 from rest_framework import serializers
 
+from normandy.recipes.api.fields import ActionImplementationHyperlinkField
 from normandy.recipes.models import Action, Recipe, RecipeAction
 
 
 class ImplementationSerializer(serializers.Serializer):
     hash = serializers.CharField(source='implementation_hash', read_only=True)
-    url = serializers.HyperlinkedIdentityField(
-        view_name='action-implementation',
-        lookup_field='name')
+    url = ActionImplementationHyperlinkField(view_name='action-implementation')
```

Comment:
I tried adding it as a class attribute like the docs suggested and I got errors about `view_name` being a required kwarg. I'll look into it more.